features/manage-position/Withdraw.tsx

```diff
@@ -0,0 +1,342 @@
+    ? Math.floor(pendingWithdrawTimeRemaining / 3600) +
+      ":" +
+      Math.floor((pendingWithdrawTimeRemaining % 3600) / 60) +
+      ":" +
+      ((pendingWithdrawTimeRemaining % 3600) % 60)
+    : null;
+  // User does not have a position yet.
+  if (collateral === null || collateral.toString() === "0") {
+    return (
+      <Container>
+        <Box py={2}>
+          <Typography>
+            <i>Create a position before withdrawing collateral.</i>
+          </Typography>
+        </Box>
+      </Container>
+    );
+  }
+
+  // User has a position and a pending withdrawal.
+  if (collateral !== null && pendingWithdraw === "Yes") {
+    return (
+      <Container>
+        <Box pt={4} pb={2}>
+          <Typography>
+            <i>You have a pending withdraw on your position!</i>
+          </Typography>
+        </Box>
+
+        <Box pb={2}>
+          <Box py={2}>
+            <Typography>
+              Once the liveness period has passed you can execute your
+              withdrawal request. You can also cancel the withdraw request if
+              you changed your mind.
+            </Typography>
+          </Box>
+          <Box py={2}>
+            <Typography>
+              <strong>Time left until withdrawal: </strong>
+              {pendingWithdrawTimeString}<br></br>
+              <strong>Requested withdrawal amount: </strong> {withdrawAmt} {collSymbol}
+            </Typography>
+          </Box>
+
+          <Box py={2}>
+            {pastWithdrawTimeStamp ? (
+              <Button
+                variant="contained"
+                onClick={handleExecuteWithdrawClick}
+                style={{ marginRight: `12px` }}
+              >{`Withdraw ${collateralToWithdraw} ${collSymbol} from your position`}</Button>
+            ) : (
+              <Button
+                variant="contained"
+                style={{ marginRight: `12px` }}
+                disabled
+              >
+                Execute Withdrawal Request
+              </Button>
+            )}
+            <Button
+              variant="contained"
+              onClick={handleCancelWithdrawClick}
+            >{`Cancel withdraw request`}</Button>
+          </Box>
+        </Box>
+      </Container>
+    );
+  }
+  // User has a position and no pending withdrawal.
+  return (
+    <Container>
+      <Box pt={4} pb={2}>
+        <Typography>
+          <i>
+            By withdrawing excess collateral from your position you will
+            decrease your collateralization ratio.
+          </i>
+        </Typography>
+      </Box>
+
+      <Box pb={2}>
+        <Box py={2}>
+          <Important>
+            IMPORTANT! Please read this carefully or you may lose money.
+          </Important>
+        </Box>
+        <Box pt={2}>
+          <Typography>
+            There are two kinds of withdrawals you can preform:
+          </Typography>
+          <ul style={{ fontSize: 16 }}>
```

Comment:
was not sure how to keep the styling the same as the `Typography` section 🤷

Reply:
`<Typography>` by default is just a `<p>` tag, this is fine too!